GliaWidgets/CallVisualizer/VisitorCode/VisitorCodeViewController.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+
+public extension CallVisualizer {
+    final class VisitorCodeViewController: UIViewController {
+        struct Props: Equatable {
+            let visitorCodeViewProps: VisitorCodeView.Props
+
+            init(visitorCodeViewProps: VisitorCodeView.Props) {
+                self.visitorCodeViewProps = visitorCodeViewProps
+            }
+        }
+
+        private let codeView = VisitorCodeView()
+
+        init(props: Props) {
+            self.props = props
+            super.init(nibName: nil, bundle: nil)
+        }
+
+        @available(*, unavailable)
+        required init?(coder: NSCoder) {
+            fatalError("init(coder:) has not been implemented")
+        }
+
+        var props: Props {
+            didSet {
+                renderProps()
+            }
+        }
+
+        public override func viewDidLoad() {
```

Comment:
the class is internal, not sure that this and `loadView` should be public.

Reply:
This wasn't a voluntary choice. Xcode throws this error after removing public from either of them.

`Overriding instance method must be as accessible as its enclosing type`

Reply:
perhaps because on line 3 you have `public extension CallVisualizer {`, let's drop `public` there also.

Reply:
fixed